java/flight/flight-jdbc-driver/src/test/java/org/apache/arrow/driver/jdbc/accessor/impl/numeric/ArrowFlightJdbcBaseIntVectorAccessorUnitTest.java

```diff
@@ -125,124 +126,104 @@ public static void setup() {
-        ArrowFlightJdbcBaseIntVectorAccessor::getObject,
-        equalTo(0xAABBCCDD));
+        ArrowFlightJdbcBaseIntVectorAccessor::getObject, equalTo(0xAABBCCDD));
   }
 
   @Test
   public void testShouldGetObjectFromTinyInt() throws Exception {
     accessorIterator.assertAccessorGetter(tinyIntVector,
-        ArrowFlightJdbcBaseIntVectorAccessor::getObject,
-        equalTo((byte) 0xAA));
+        ArrowFlightJdbcBaseIntVectorAccessor::getObject, equalTo((byte) 0xAA));
   }
 
   @Test
   public void testShouldGetObjectFromSmallInt() throws Exception {
     accessorIterator.assertAccessorGetter(smallIntVector,
-        ArrowFlightJdbcBaseIntVectorAccessor::getObject,
-        equalTo((short) 0xAABB));
+        ArrowFlightJdbcBaseIntVectorAccessor::getObject, equalTo((short) 0xAABB));
   }
 
   @Test
   public void testShouldGetObjectFromBigInt() throws Exception {
     accessorIterator.assertAccessorGetter(bigIntVector,
-        ArrowFlightJdbcBaseIntVectorAccessor::getObject,
-        equalTo(0xAABBCCDDEEFFAABBL));
+        ArrowFlightJdbcBaseIntVectorAccessor::getObject, equalTo(0xAABBCCDDEEFFAABBL));
   }
 
   @Test
   public void testShouldGetObjectFromUnsignedInt() throws Exception {
     accessorIterator.assertAccessorGetter(int4Vector,
-        ArrowFlightJdbcBaseIntVectorAccessor::getObject,
-        equalTo(0x80000001));
+        ArrowFlightJdbcBaseIntVectorAccessor::getObject, equalTo(0x80000001));
   }
 
   @Test
   public void testShouldGetObjectFromIntVectorWithNull() throws Exception {
     accessorIterator.assertAccessorGetter(intVectorWithNull,
-        ArrowFlightJdbcBaseIntVectorAccessor::getObject,
-        CoreMatchers.nullValue());
+        ArrowFlightJdbcBaseIntVectorAccessor::getObject, CoreMatchers.nullValue());
   }
 
   @Test
   public void testShouldGetBigDecimalFromIntVectorWithNull() throws Exception {
     accessorIterator.assertAccessorGetter(intVectorWithNull,
-        ArrowFlightJdbcBaseIntVectorAccessor::getBigDecimal,
-        CoreMatchers.nullValue());
+        ArrowFlightJdbcBaseIntVectorAccessor::getBigDecimal, CoreMatchers.nullValue());
   }
 
   @Test
   public void testShouldGetBigDecimalWithScaleFromIntVectorWithNull() throws Exception {
-    accessorIterator
-        .assertAccessorGetter(intVectorWithNull, accessor -> accessor.getBigDecimal(2),
-            CoreMatchers.nullValue());
+    accessorIterator.assertAccessorGetter(intVectorWithNull, accessor -> accessor.getBigDecimal(2),
+        CoreMatchers.nullValue());
   }
 
-  @Test
+  @Test(expected = SQLException.class)
   public void testShouldGetBytesFromSmallVector() throws Exception {
     byte[] value = new byte[] {(byte) 0xaa, (byte) 0xbb};
-
-    accessorIterator
-        .assertAccessorGetter(smallIntVector, ArrowFlightJdbcBaseIntVectorAccessor::getBytes,
-            CoreMatchers.is(value));
+    accessorIterator.assertAccessorGetter(smallIntVector,
+        ArrowFlightJdbcBaseIntVectorAccessor::getBytes, CoreMatchers.is(value));
```

Comment:
Change this to use the new assert method that throws an exception

Reply:
fixed